fdbclient/NativeAPI.actor.cpp

```diff
@@ -996,6 +996,21 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 				throw invalid_option_value();
 			}
 			break;
+		case FDBNetworkOptions::CLIENT_BUGGIFY_ENABLE:
+			enableBuggify(true, BuggifyType::Client);
+			break;
+		case FDBNetworkOptions::CLIENT_BUGGIFY_DISABLE:
+			enableBuggify(false, BuggifyType::Client);
+			break;
+		case FDBNetworkOptions::CLIENT_BUGGIFY_SECTION_ACTIVATED_PROBABILITY:
+			validateOptionValue(value, true);
+			clearBuggifySections(BuggifyType::Client);
+			P_BUGGIFIED_SECTION_ACTIVATED[int(BuggifyType::Client)] = 100.0/double(extractIntOption(value, 0, 100));
```

Comment:
Should this be `value/100.0` rather than `100.0/value`? The same question applies to `CLIENT_BUGGIFY_SECTION_FIRED_PROBABILITY` as well.

Reply:
Oh yes, of course. This is an embarrassing one